Stdlib modulemd updates (#18915)
ballerina-platform/ballerina-lang: stdlib/time/src/main/ballerina/src/time/Module.md

    io:println(nanoTime);
    string zoneId = time.zone.id;    // Time zone as an identifier. E.g., “America/Panama”.
    io:println(zoneId);
    int zoneoffset = time.zone.offset;    // Time zone as an offset. E.g., -05:00.
    io:println(zoneoffset);

    // Get the current date and time upto milliseconds.
    int year = time:getYear(time);    // E.g., 2018
    io:println(year);
    int month = time:getMonth(time);    // E.g., 3
    io:println(month);
    int day = time:getDay(time);    // E.g., 1
    io:println(day);
    int hour = time:getHour(time);    // E.g., 18
    io:println(hour);
    int minute = time:getMinute(time);    // E.g., 56
    io:println(minute);
    int second = time:getSecond(time);    // E.g., 23
    io:println(second);
    int milliSecond = time:getMilliSecond(time);    // E.g., 555
    io:println(milliSecond);
    string weekday = time:getWeekday(time);// Day of the week. E.g., “TUESDAY”.
    io:println(weekday);
}
```

### Creating a time/date record

```ballerina
import ballerina/io;
import ballerina/time;

public function main() {

>     // Create a record of type ‘Time’ with time zone.

Reviewer: ```suggestion
    // Creates a record of the ‘Time’ type with the  time zone.
```

    time:TimeZone zoneIdValue = {id: "America/Panama"};
    time:Time time1 = {time: 1498488382000, zone: zoneIdValue};
    io:println(time1);

>     // Create a record of type ‘Time’ with the time zone offset.

Reviewer: ```suggestion
    // Creates a record of the ‘Time’ type with the time zone offset.
```

    time:TimeZone zoneOffsetValue = {id: "-05:00"};
    time:Time time2 = {time: 1498488382000, zone: zoneOffsetValue};
    io:println(time2);

>     // Create a record of type ‘Time’ without the time zone.

Reviewer: ```suggestion
    // Creates a record of the ‘Time’ type without the time zone.
```

    time:TimeZone noZoneValue = {id: ""};
    time:Time time3 = {time: 1498488382000, zone: noZoneValue};
    io:println(time3);

>     // Create a record of type ‘Time’ with time and date. E.g., 2018-03-28T23:42:45.554-05:00

Reviewer: ```suggestion
    // Creates a record of the ‘Time’ type with time and date. E.g., 2018-03-28T23:42:45.554-05:00
```

    time:Time | error dateTime = time:createTime(2018, 3, 28, 23, 42, 45, 554, "America/Panama");
    if (dateTime is time:Time) {
        io:println(dateTime);
    } else {
        io:println("An error occurred: ", dateTime);
    }
}
```

### Formatting a time/date to a string

```ballerina
import ballerina/io;
import ballerina/time;

public function main() returns error? {
    time:TimeZone zoneValue = {id: "America/Panama"};
    time:Time time = {time: 1498488382444, zone: zoneValue};

>     //Format a time to a string of a given pattern.

Reviewer: ```suggestion
    //Formats a time to a string of a given pattern.
```

    string | error time1 = check time:format(time, "yyyy-MM-dd'T'HH:mm:ss.SSSZ");    //E.g., “2017-06-26T09:46:22.444-0500”.
    io:println(time1);

>     //Format a time to a string of the RFC-1123 format.

Reviewer: ```suggestion
    //Formats a time to a string of the RFC-1123 format.
```

    string | error time2 = time:format(time, time:TIME_FORMAT_RFC_1123);    // E.g., "Mon, 26 Jun 2017 09:46:22 -0500”
    io:println(time2);

>     // Convert a time record to a string value.

Reviewer: ```suggestion
    // Converts a time record to a string value.
```

    string time3 = time:toString(time);//”2017-06-26T09:46:22.444-05:00”
    io:println(time3);
}
```

### Parsing a string to time/date

```ballerina
import ballerina/io;
import ballerina/time;

public function main() returns error? {

>     // Parse a time string of a given format.

Reviewer: ```suggestion
    // Parses a time string of a given format.
```

    time:Time | error time1 = time:parse("2017-06-26T09:46:22.444-0500", "yyyy-MM-dd'T'HH:mm:ss.SSSZ");    // The ‘Z’ stands for the time zone.
    io:println(time1);

>     // Parse a time string of the RFC-1123 format.

Reviewer: ```suggestion
    // Parses a time string of the RFC-1123 format.
```

    time:Time | error time2 = time:parse("Wed, 28 Mar 2018 11:56:23 +0530", time:TIME_FORMAT_RFC_1123);
    io:println(time2);
}
```

### Setting time durations

```ballerina
import ballerina/time;

public function main() {

>     // Add a duration to a given time.

Reviewer: ```suggestion
    // Adds a duration of a given time.
```

    time:Time | error time1 = time:parse("2017-06-26T09:46:22.444-0500", "yyyy-MM-dd'T'HH:mm:ss.SSSZ");
    if (time1 is time:Time) {
        time1 = time:addDuration(time1, 1, 1, 1, 1, 1, 1, 1);
    // Adds 1 year, 1 month, 1 day, 1 hour, 1 minute, 1 second, and 1 millisecond.
    }

>     // Subtract a duration from a given time.

Reviewer: ```suggestion
    // Subtracts a duration from a given time.
```

    time:Time | error time2 = time:parse("2016-03-01T09:46:22.444-0500", "yyyy-MM-dd'T'HH:mm:ss.SSSZ");
    if (time2 is time:Time) {
        time2 = time:subtractDuration(time2, 1, 1, 1, 1, 1, 1, 1);
    // Subtracts 1 year, 1 month, 1 day, 1 hour, 1 minute, 1 second, and 1 millisecond.
    }

>     // Get the duration between two times.

Reviewer: ```suggestion
    // Gets the duration between two times.
```
